content/syllabuses/data-eng-part-1.md

```diff
@@ -7,6 +7,7 @@ _db_id: 4
 - {{< contentlink path="agile/agile-scrum" >}}
 - {{< contentlink path="topics/clean-code/python" >}}
 - {{< contentlink path="topics/file-and-directory-naming/python" >}}
+- {{< contentlink path="syllabus-written-assignments/file-directory-naming" >}}
```

Comment:
Once upon a time we separated everything out into topic and project directories because we didn't have good frontmatter yet. That is technical debt that needs to be overcome. We shouldn't imitate technical debt.

Syllabus content should be thought of as code. Good code should be cohesive. 

The file and directory naming assignment should be with the file and directory naming content. It shouldn't be super far away. The things that are used together should be together.